tests/resources/stages/Tokenizer/metadata/part-00000

```diff
@@ -1 +1 @@
-{"class":"org.apache.spark.ml.feature.Tokenizer","timestamp":1588797806167,"sparkVersion":"2.4.0","uid":"Tokenizer_7e123b887c49","paramMap":{"outputCol":"words","inputCol":"text"},"defaultParamMap":{"outputCol":"Tokenizer_7e123b887c49__output"}}
+{"class":"org.apache.spark.ml.feature.Tokenizer","timestamp":1589129933974,"sparkVersion":"2.4.5","uid":"Tokenizer_1bfbddaa18be","paramMap":{"outputCol":"words","inputCol":"text"},"defaultParamMap":{"outputCol":"Tokenizer_1bfbddaa18be__output"}}
```

Comment:
probably this file should not be pushed

Reply:
I have a TODO to delete files created in the tests